src/date/DateTBody.jsx

```diff
@@ -208,7 +209,14 @@ const DateTBody = React.createClass({
             title={getTitleString(current)} className={cls}
           >
             {dateHtml}
-          </td>);
+          </td>
+        );
+
+        if (cellRender) {
+          dateCells.push(cellRender(cell));
```

Comment:
The API's arguments are different from other render API https://github.com/react-component/calendar